docs/_data/mesheryctlcommands/lifecycle/system/configure/command/config.yaml

```diff
@@ -1,4 +1,4 @@
 name: config
 description: Configures Meshery to use a Kubernetes cluster.
```

Comment:
What dependencies must users have installed on their system in order to execute this command?


// @hexxdump